src/ui/Helpers/ViewModelHelpers.cs

```diff
@@ -11,23 +11,14 @@ public static class ViewModelHelpers
     {
         public static string GetCourseVariantType(this Course course)
         {
-            var result = string.IsNullOrWhiteSpace(course.ProfpostFlag) ? "QTS " : "PGCE with QTS ";
+            var result = string.IsNullOrWhiteSpace(course.ProfpostFlag) ? "QTS, " : "PGCE with QTS, ";
```

Comment:
The comma should only be there for the full or part time variant. Not the others.

Reply:
Done